packages/react-storage/src/components/StorageBrowser/__tests__/createStorageBrowser.spec.tsx

```diff
@@ -19,4 +20,6 @@ describe('createStorageBrowser', () => {
       expect(render(<StorageBrowser />).container).toBeDefined();
     });
   });
+
+  it.todo('throws when registerAuthListener is not a function');
```

Comment:
Info: when would this test case be returned to? Should it not be within this set of changes?